handshake_docs/handshake_flow.md

```diff
@@ -0,0 +1,35 @@
+# Handshake Flow for DIDComm Mediator-rs
+
+## 1. Communication Initiation
+### Participant Identification:
+- Each mobile agent generates a DID (Decentralized Identifier).
+
+- Agents exchange their DIDs to identify each other. The initial exchange might involve sending a DIDComm message that includes the agent’s DID and a nonce (a random value used for security purposes).
```

Comment:
This exchange occurs in the of an OOB message. Generally in which one edge agent scans the QR code of another edge agent.